docs/en/connector-v2/sink/LocalFile.md

```diff
@@ -166,6 +167,10 @@ When File Format is Excel,The maximum number of data items that can be cached in
 
 Writer the sheet of the workbook
 
+### enable_header_write [boolean]
+
+false:dont write header,true:write header.
```

Comment:
It seems you only implement this in text format, if so it's needed to mention this only support in text format.

Reply:
yeah，i will fix it

Reply:
support text and csv